src/components/pages/DAOcreator/index.tsx

```diff
@@ -198,11 +223,58 @@ class DAOcreator extends React.Component<Props, State> {
       }
     };
 
+    const PreviewDialog = () => (
+      <Dialog
+        open={recoverPreviewOpen}
+        fullScreen={true}>
+        <DialogTitle id="simple-dialog-title">
+          Preview
+      </DialogTitle>
+        <DialogContent>
+          <ReviewStep
```

Comment:
The review step as it exists now renders a "change this section" gear icon next to each section. Let's change ReviewStep's props to make `setStep` an optional prop, and if it isn't present it doesn't render the `modifyStep(...)` component on line 53.